app/serializers/loan_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class LoanSerializer < ActiveModel::Serializer
+  attributes :id, :borrower_id, :lender_id, :amount,
+             :interest, :purpose, :repayment_till, :expected_return,
+             :status, :total_return
+
+  # Total Return Principle Amount + Interest
+  def total_return
```

Comment:
If you plan to use this method somewhere in the code, then move it outside in another file (like a module) and use it here. else, there is no need to change.